TestingDangerApp/ViewController.swift

```diff
@@ -1,3 +1,17 @@
+// Comentários
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Edu15/TestingDangerApp/blob/673f6e5d02294f7e921a1c844860d76417ab39c5/TestingDangerApp/ViewController.swift#L14"></span>Limit vertical whitespace to a single empty line. Currently 2.<br />
<code>vertical_whitespace</code> <code>ViewController.swift:14</code></td>
    </tr>
  </tbody>
</table>